### PR TITLE
Add HttpServer.setBodyAsBytes

### PR DESCRIPTION
**Pull request by blaix**
Converting bytes to string results in corrupt images. This lets you explicitly set the body to bytes.

Updated examples/http-server to serve /george.jpeg

### CODE REVIEW

File: src/Gren/Kernel/HttpServer.js [hunk 1]
```diff
@@ -85,6 +85,11 @@ var _HttpServer_setBody = F2(function (body, res) {
   return res;
 });
 
+var _HttpServer_setBodyToBytes = F2(function (bytes, res) {
+  let buffer = Buffer.from(bytes.buffer);
```
Comment (robinheghan) [inline src/Gren/Kernel/HttpServer.js:89]: It's better to use `Uint8Array` here.

For more info: https://sindresorhus.com/blog/goodbye-nodejs-buffer

File: src/HttpServer/Response.gren [hunk 2]
```diff
@@ -73,12 +81,26 @@ setHeader key value (Response responseKey response) =
...
+setBodyToString body (Response key response) =
     Response key
-        { response | body = body }
+        { response | body = StringBody body }
+
+
+{-| Set the body of the response to some bytes.
+-}
+setBodyToBytes : Bytes -> Response -> Response
+setBodyToBytes body (Response key response) =
```
Comment (robinheghan) [inline src/HttpServer/Response.gren:101]: Bear in mind that English is not my native language, but would `setBodyAsBytes` be better? 🤔

**Review by robinheghan — CHANGES_REQUESTED** [review]
Love the picture!

Overall, I think this looks good. Just one code change I'd like you to do, and one question regarding naming, then we're good to go.

File: src/Gren/Kernel/HttpServer.js (hunk 1 above)
Comment (blaix) [reply src/Gren/Kernel/HttpServer.js]: Good to know, thanks!

File: src/HttpServer/Response.gren (hunk 2 above)
Comment (blaix) [reply src/HttpServer/Response.gren]: I think so, yes.

**Review by robinheghan — APPROVED** [review]
Will merge and cut a new release later tonight.

Thanks!

**Pull request merged**